dj/api/helpers.py

```diff
@@ -46,3 +46,20 @@ def get_database_by_name(session: Session, name: str) -> Database:
             http_status_code=404,
         )
     return database
+
+
+def get_column(node: NodeRevision, column_name: str) -> Column:
+    """
+    Get a column from a node revision
+    """
+    requested_column = None
+    for node_column in node.columns:
+        if node_column.name == column_name:
+            requested_column = node_column
```

Comment:
nit: I don't remember us doing this kind of optimization across the code, but I would have added `break` here if I was writing this.

Reply:
Great idea, I added in a `break`.